src/dataworker/Dataworker.ts

```diff
@@ -11,16 +22,21 @@ export class Dataworker {
     readonly logger: winston.Logger,
     readonly spokePoolClients: { [chainId: number]: SpokePoolClient },
     readonly hubPoolClient: HubPoolClient,
-    readonly multiCallerClient: MultiCallerClient | any
+    readonly multiCallerClient: MultiCallerClient | any,
+    readonly maxRefundsPerLeaf: number = 25
```

Comment:
```suggestion
    readonly maxRefundsPerLeaf: number = 25 // TODO: this will be removed in the future when the `ConfigStoreClient` has been implemented.
```

this is the perfect example of where we will use the `ConfigStore` contract: it will contain this exact kind of setting, as well as other settings for defining how bundles are built.

what you have for now is fine but we should just indicate that this is going to change in the future.

Reply:
I'm going to create a stand-in `ConfigStoreClient` actually with a hardcoded `maxRefundsPerLeaf` class var that we can replace later